src/conn/conn_api.c

```diff
@@ -592,7 +592,6 @@ __conn_get_page_log(WT_CONNECTION *wt_conn, const char *name, WT_PAGE_LOG **page
     TAILQ_FOREACH (npage_log, &conn->pagelogqh, q)
         if (WT_STREQ(npage_log->name, name)) {
             page_log = npage_log->page_log;
-            WT_RET(page_log->pl_add_reference(page_log));
```

Comment:
I don't think this is correct. If a page log handle references a page log, we need to track that relationship and avoid freeing the page log until all handles are closed.